<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MedusaRsrc.qrc">
    <normaloff>:/images/Gear.png</normaloff>:/images/Gear.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="general">
      <attribute name="icon">
       <iconset resource="MedusaRsrc.qrc">
        <normaloff>:/images/Gear.png</normaloff>:/images/Gear.png</iconset>
      </attribute>
      <attribute name="title">
       <string>General</string>
      </attribute>
      <widget class="QGroupBox" name="textGroupBox">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>611</width>
         <height>61</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Text</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <layout class="QFormLayout" name="textFormLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="labelFont">
            <property name="text">
             <string>Font</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="FontPicker" name="textFont">
            <property name="enabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="color">
      <attribute name="icon">
       <iconset resource="MedusaRsrc.qrc">
        <normaloff>:/images/Palette.png</normaloff>:/images/Palette.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Color</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>172</width>
            <height>501</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Disassembly view:</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <property name="sizeConstraint">
               <enum>QLayout::SetMinimumSize</enum>
              </property>
              <item>
               <layout class="QFormLayout" name="formLayout_3">
                <property name="sizeConstraint">
                 <enum>QLayout::SetMinimumSize</enum>
                </property>
                <item row="0" column="0">
                 <widget class="QLabel" name="backgroundListingColorLabel">
                  <property name="text">
                   <string>Background</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="ColorPicker" name="medusaBackground">
                  <property name="text">
                   <string>#FF1969</string>
                  </property>
                  <property name="color" stdset="0">
                   <color>
                    <red>255</red>
                    <green>25</green>
                    <blue>105</blue>
                   </color>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="title">
                 <string>Instruction:</string>
                </property>
                <property name="flat">
                 <bool>false</bool>
                </property>
                <property name="checkable">
                 <bool>false</bool>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <property name="sizeConstraint">
                  <enum>QLayout::SetMinimumSize</enum>
                 </property>
                 <item>
                  <layout class="QFormLayout" name="formLayout">
                   <property name="sizeConstraint">
                    <enum>QLayout::SetMinimumSize</enum>
                   </property>
                   <property name="fieldGrowthPolicy">
                    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                   </property>
                   <item row="0" column="0">
                    <widget class="QLabel" name="mnemonicColorLabel">
                     <property name="layoutDirection">
                      <enum>Qt::LeftToRight</enum>
                     </property>
                     <property name="text">
                      <string>Mnemonic</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0">
                    <widget class="QLabel" name="registerColorLabel">
                     <property name="text">
                      <string>Register</string>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="0">
                    <widget class="QLabel" name="immediateColorLabel">
                     <property name="text">
                      <string>Immediate</string>
                     </property>
                    </widget>
                   </item>
                   <item row="3" column="0">
                    <widget class="QLabel" name="commentColorLabel">
                     <property name="text">
                      <string>Comment</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1">
                    <widget class="ColorPicker" name="registerColor"/>
                   </item>
                   <item row="0" column="1">
                    <widget class="ColorPicker" name="mnemonicColor"/>
                   </item>
                   <item row="2" column="1">
                    <widget class="ColorPicker" name="immediateColor"/>
                   </item>
                   <item row="3" column="1">
                    <widget class="ColorPicker" name="commentColor"/>
                   </item>
                   <item row="4" column="0">
                    <widget class="QLabel" name="selectionColorLabel">
                     <property name="text">
                      <string>Selection</string>
                     </property>
                    </widget>
                   </item>
                   <item row="4" column="1">
                    <widget class="ColorPicker" name="selectionColor"/>
                   </item>
                   <item row="5" column="0">
                    <widget class="QLabel" name="operationColorLabel">
                     <property name="text">
                      <string>Operator</string>
                     </property>
                    </widget>
                   </item>
                   <item row="6" column="0">
                    <widget class="QLabel" name="keywordColorLabel">
                     <property name="text">
                      <string>Keyword</string>
                     </property>
                    </widget>
                   </item>
                   <item row="5" column="1">
                    <widget class="ColorPicker" name="operatorColor"/>
                   </item>
                   <item row="6" column="1">
                    <widget class="ColorPicker" name="keywordColor"/>
                   </item>
                   <item row="7" column="0">
                    <widget class="QLabel" name="labelColorLabel">
                     <property name="text">
                      <string>Label</string>
                     </property>
                    </widget>
                   </item>
                   <item row="8" column="0">
                    <widget class="QLabel" name="stringColorLabel">
                     <property name="text">
                      <string>String</string>
                     </property>
                    </widget>
                   </item>
                   <item row="7" column="1">
                    <widget class="ColorPicker" name="labelColor"/>
                   </item>
                   <item row="8" column="1">
                    <widget class="ColorPicker" name="stringColor"/>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="title">
                 <string>Address view:</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <property name="sizeConstraint">
                  <enum>QLayout::SetMinimumSize</enum>
                 </property>
                 <item>
                  <layout class="QFormLayout" name="formLayout_2">
                   <property name="sizeConstraint">
                    <enum>QLayout::SetMinimumSize</enum>
                   </property>
                   <item row="0" column="0">
                    <widget class="QLabel" name="backgroundAddressColorLabel">
                     <property name="text">
                      <string>Background</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="ColorPicker" name="addressBackground"/>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <spacer name="verticalSpacer_4">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorPicker</class>
   <extends>QLineEdit</extends>
   <header>colorpicker/ColorPicker.h</header>
  </customwidget>
  <customwidget>
   <class>FontPicker</class>
   <extends>QLineEdit</extends>
   <header>fontpicker/FontPicker.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MedusaRsrc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>369</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
